src/Simulation/QSharpFoundation/Convert/Convert.qs

```diff
@@ -164,7 +164,21 @@ namespace Microsoft.Quantum.Convert {
     /// the mappings of each Pauli operator in big-endian order
     /// `bits(Pn) ... bits(P0)`.
     function PauliArrayAsInt(paulis : Pauli[]) : Int {
-        body intrinsic;
```

Comment:
Now that this is no longer `body intrinsic` could you remove the corresponding C# implementation from https://github.com/microsoft/qsharp-runtime/blob/main/src/Simulation/QSharpFoundation/Convert/Convert.cs#L162 ? That will ensure that the C# tests cover this implementation instead of getting overridden.

Reply:
Removed. Thanks.